spring-cloud-loadbalancer/src/test/java/org/springframework/cloud/loadbalancer/core/ServiceInstanceListSuppliersTestUtils.java

```diff
@@ -58,4 +59,12 @@ private ServiceInstanceListSuppliersTestUtils() {
 		});
 	}
 
+	private static String getUri(ServiceInstance serviceInstance, String healthCheckPath) {
```

Comment:
Let's make the method `getUri` in `ServiceInstanceListSupplierBuilder` package-private and reuse it in this test rather than duplicating the code in the test.